Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM openmicroscopy/ome-files-cpp-u1604
```

Comment:
Given the time and resource constraints, working on the benchmarking scripts using Docker was the most convenient to get something working on `beluga` and get some initial numbers on both platforms for both Linux/C++. For the final metrics, certainly open to discussing the various options between `Windows/bare metal` vs `Linux/bare metal` vs `Linux/Docker`.

Independently of the choice above, releasing versioned images of our tools bundled a set of benchmarking scripts might simplify the future development of our libraries.